lib/internal/event_target.js

```diff
@@ -388,6 +403,31 @@ class Listener {
     }
   }
 
+  get once() {
```

Comment:
Honestly I'm not really sure how to articulate the preference. It's something I've seen done with flag properties before but I can't find a justification other than "it'd look more familiar to me based on other code I've seen".

Reply:
I like this style because it makes the properties accessible the same way they were before, but yeah, ultimately no strong preferences here from my side, I was just curious.

Reply:
I was honestly convinced I had a good reason before you asked so thanks for asking and making me think about it. Very easy to fall into familiar patterns.